dna/convert_test.go

```diff
@@ -36,6 +36,40 @@ func TestBasesToString(t *testing.T) {
 	}
 }
 
+var equalStringAndBase = []struct {
+	character string
+	base      Base
+}{
+	{"A", A},
+	{"C", C},
+	{"G", G},
+	{"T", T},
+	{"N", N},
+	{"a", LowerA},
+	{"c", LowerC},
+	{"g", LowerG},
+	{"t", LowerT},
+	{"n", LowerN},
+}
+
+func TestStringToBase(t *testing.T) {
+	for _, test := range equalStringAndBase {
+		actual := StringToBase(test.character)
+		if actual != test.base {
+			t.Errorf("StringToBase(%s): expected %d, actual %d", test.character, test.base, actual)
```

Comment:
I was wondering if there was a reason to print the "test.character" here in this error message instead of using the "actual" variable created on line 57.